mlflow/server/js/src/experiment-tracking/components/ExperimentListView.tsx

```diff
@@ -230,19 +228,21 @@ export class ExperimentListView extends Component<Props, State> {
 
   render() {
     const { hidden } = this.state;
+    const { activeExperimentIds, designSystemThemeApi } = this.props;
```

Comment:
@hubertzub-db is this the right way to use HoC (what does it stand for btw)?

Reply:
yeah, it works! thanks!